src/components/MapEditor/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { memo, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Map as MapComponent, Marker } from '@datapunt/react-maps';
+import { ViewerContainer } from '@datapunt/asc-ui';
+import { Zoom } from '@datapunt/amsterdam-react-maps/lib/components';
+import styled from '@datapunt/asc-core';
+import BackgroundLayer from 'components/BackgroundLayer';
+import { markerIcon } from 'shared/services/configuration/map-markers';
+import { feature2location } from 'shared/services/map-location';
+
+const MapWrapper = styled.div`
+  position: relative;
+`;
+
+const StyledMap = styled(MapComponent)`
+  width: 100%;
+  height: 450px;
```

Comment:
What happens when we need an instance of `MapEditor` to cover a different height?

Reply:
Good point! I have changed this component to pass the styles from through. It can be used now as a styled-component.